### ISSUE TITLE
custom icon color set as background color

### ISSUE THREAD

**Issue by bes-internal** [caroso1222/notyf#45]
https://github.com/caroso1222/notyf/blob/21cee095f5b0163907ec02242852823c456ac0a5/src/notyf.view.ts#L94
It turns out that the icon becomes the background color. It should be the color of the text or set as a separate options.

**Comment by caroso1222** [comment]
Thanks for noting it. I'll take care of that in the next release.

**Comment by luongvanphuc** [comment]
+1
I have to override the style manually:
```
.notyf__icon .icon {
  color: #ffffff !important;
}
```

**Comment by caroso1222** [comment]
I've been working on this lately. As you all mentioned, I realized this was a poor design choice from the beginning. I can't change the current behavior, though, as it would break the default functionality for a lot of people: they are changing the toast background and they expect the default icon to change accordingly.

That being said, the only workaround I can see here is to allow users to change the icon color like this:

```javascript
notyf.success({
  message: 'Here is a message',
  icon: {
    className: 'fa fa-check',
    color: '#ffffff',
  },
});
```

I've opened a PR to add this but I'll need to sleep on it for 1 or 2 days. Thanks all for your input!
